src/main/java/com/Novi/TechItEasy/services/TelevisionService.java

```diff
@@ -145,6 +151,85 @@ public TelevisionDto changeTelevision(Long id, TelevisionInputDto television) {
     }
 
 
+    ///// For assigning remote controllers to television by id /////
+    public TelevisionDto assignRemoteControllerToTelevision(Long televisionId, IdInputDto remoteId) {
+        Optional<Television> fetchedTelevision =  televisionRepository.findById(televisionId);
+        Optional<RemoteController> fetchedRemoteController = remoteControllerRepository.findById(remoteId.id);
+
+        if (fetchedTelevision.isPresent()) {
+            if (fetchedRemoteController.isPresent()) {
+
+                fetchedTelevision.get().setRemoteController(fetchedRemoteController.get());
```

Comment:
Je kunt de uitkomst van dit statement beter opslaan in een variabel, of direct doorgeven als input van de save methode.